### PR TITLE
Added new example for Scara arm.

### PR DESCRIPTION
**Pull request by Fernandohf**
Added new example to solve Kinematics and Dynamics of Robotic Arm.

Example
-------

   - [x] There are no merge conflicts.
   - [x] A figure (preferrably SVG) has been added for the example. NOT SVG.
   - [x] The required versions of all dependencie have been noted. (For
     notebooks, the [version information
     extension](https://github.com/jrjohansson/version_information) is helpful
     for this.)
   - [x] A `run.py` file that executes the example is included.
   - [x] All reviewer comments have been addressed.

### CODE REVIEW

**Comment by moorepants** [comment]
This looks like a nice addition. Thank you. All of the Travis tests are failing due to the whitespace check. Can you please run linter on the python files and remove any trailing whitespace?

```
The command "(flake8 --select W1,W2,W3 --exclude booger . && flake8 --select F401 --exclude booger,__init__.py,conf.py . && flake8 --select W2,W3 --filename='*.js' examples/ pydy/viz/static/js/dyviz/) || (echo "Please run a linter and fix errors, e.g. pylint, flake8, pep8, etc."; false)" exited with 1.
```

File: .gitignore [hunk 1]
```diff
@@ -53,3 +53,5 @@ nosetests.xml
 
 # vim swap files
 *.swp
+examples/scara_arm/.vscode/.ropeproject/objectdb
+examples/scara_arm/.vscode/.ropeproject/config.py
```
Comment (moorepants) [inline .gitignore]: This seems too specific. Please do not add these to the .gitignore. Maybe there are some more general statements for vscode that would be more appropriate.

File: examples/scara_arm/README.md [hunk 2]
```diff
@@ -0,0 +1,36 @@
+# Scara Robot - Kinetics and Dynamics Equations
+
+![Robot](img\2dscara.png)
```
Comment (moorepants) [inline examples/scara_arm/README.md]: forward slash should be a backslash, or better yet, don't put the image in a subdirectory.

File: .gitignore (hunk 1 above)
Comment (Fernandohf) [reply .gitignore]: Changed. Now it will ignore VS Code local settings.

File: examples/scara_arm/README.md (hunk 2 above)
Comment (Fernandohf) [reply examples/scara_arm/README.md]: The image subdirectory was removed.

**Comment by moorepants** [comment]
Note that I have a changed submitted to sympy to correct the failing tests: https://github.com/sympy/sympy/pull/15423. Let me see if I can get that in quickly.

**Comment by moorepants** [comment]
I'm going to go ahead and merge this. I'm not sure with the downstream code will be fixed. Thank you for the contribution.

**Pull request merged**